home/modules/contribute/pages/extensions.adoc

```diff
@@ -0,0 +1,39 @@
+= Site Extensions
+
+Test and document your Asciidoctor.js or Antora extensions here.
+
+== inline JIRA macro
+
+=== Usage
+
+[source,asciidoc]
+----
+== Fixed issues:
+
+* jira:JSCBC-1234[]: reversed the polarity of the neutron flow
+* jira:AV-2345[]: frobnicated the gostaks
+* jira:DOC-999[The dreaded DOC-999 ticket...]: implemented jira macro
+----
+
+[,yml]
```

Comment:
```suggestion
[source,yml]
```

Reply:
FYI: The `source` style is now inferred if a language is specified.

Reply:
Good to know - though I'm happy with keeping it explicit personally, let's discuss within the team.